style.css

```
@@ -0,0 +1,258 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: none;
+  box-sizing: border-box;
+  background: transparent;
+  font-family: inherit;
+}
+
+a {
+  text-decoration: none;
+}
+
+a img {
+  border: none;
+}
+
+button,
+a {
+  cursor: pointer;
+  color: inherit;
+}
+
+textarea {
+  resize: none;
+}
+
+button,
+textarea,
+select {
+  appearance: none !important;
+  border-radius: 0;
+}
+
+/*End of CSS reset*/
+
+body {
+  width: 100%;
+  height: 100%;
+  line-height: 1.6;
+  font-family: "Russo One", sans-serif;
+  color: #757973ee;
+}
+
+.background {
+  position: relative;
+}
+
+.background::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-image: url("../1767974.jpg");

```

"próba comment :)"
```suggestion
  background-image: url("1767974.jpg");

```
itt ha ezt írod megjelenik a háttérkép